### PR TITLE
Improve choosing plugins/dependencies in init script #358

### PR DESCRIPTION
**Pull request by tristanisham**
### Check List

- [x] Have you read the
      [CODE OF CONDUCT](https://github.com/lumeland/lume/blob/master/CODE_OF_CONDUCT.md)
- [x] Have you read the document
      [CONTRIBUTING](https://github.com/lumeland/lume/blob/master/CONTRIBUTING.md)

      - [ ] One pull request per feature. If you want to do more than one thing,
      send multiple pull request.
      - [ ] Write tests.
      - [ ] Run deno `fmt` to fix
      the code format before commit.
      - [ ] Document any change in the
      `CHANGELOG.md`.

### CODE REVIEW

**Comment by tristanisham** [comment]
Unfortunately, this does change the appearance of the cli init script's plugin chooser (had to remove the URLs) for temporary compatibility. But this can be addressed in a future update. This choice only adds one dependency and doesn't require any new functions.

**Comment by oscarotero** [comment]
Hi, thanks for this.
I like the new interface but I don't like it requires a new dependency (specially a NPM dependency).
I think a similar output can be achieved with Cliffy (the library I'm already using to build the CLI).

Take a look to this: https://cliffy.io/docs@v0.25.7/prompt

Or better, run this script:

```
deno run https://deno.land/x/cliffy@v0.25.7/examples/prompt/prompt_list.ts
```

**Review by oscarotero — CHANGES_REQUESTED** [review]
I think it's better to use the prompt function from Cliffy (https://cliffy.io/docs@v0.25.7/prompt)

**Comment by tristanisham** [comment]
@oscarotero Updated to use the existing cliffy dependency. I also removed inquirer.

File: cli/init.ts [hunk 1]
```diff
@@ -23,12 +23,12 @@ async function initConfig(): Promise<string[] | undefined> {
   }
 
   // Generate the code for the config file
-  const code = [`import lume from "lume/mod.ts";`];
+  const code = [`import lume from "https://deno.land/x/lume/mod.ts";`];
```
Comment (oscarotero) [inline cli/init.ts:26]: The URL here must be `lume/mod.ts`. It's resolved with the import map.

File: cli/init.ts [hunk 2]
```diff
@@ -23,12 +23,12 @@ async function initConfig(): Promise<string[] | undefined> {
...
 
-  const plugins = getPlugins();
+  const plugins = await getPlugins();
   plugins.forEach((name) =>
     code.push(
-      `import ${name} from "lume/plugins/${name}.ts";`,
+      `import ${name} from "https://deno.land/x/lume/plugins/${name}.ts";`,
```
Comment (oscarotero) [inline cli/init.ts:31]: Same here, it must be `lume/plugins/${name}.ts`

**Review by oscarotero — CHANGES_REQUESTED** [review]

**Comment by oscarotero** [comment]
Thank you. Much better now!
I just let a couple of comments related to the URL to import the lume modules. You have changed the bare imports to full URLs, not sure why.

**Comment by tristanisham** [comment]
@oscarotero I've changed it back to using the import map URLs. 👍🏼

**Comment by oscarotero** [comment]
Thanks! The tests are failing due the code format. Can you run `deno fmt` and commit again?

**Review by oscarotero — APPROVED** [review]

**Comment by tristanisham** [comment]
Updated the format and removed unused imports.

**Comment by oscarotero** [comment]
Thank you!

**Comment by tristanisham** [comment]
@oscarotero You're welcome! This was a really nice experience!

**Pull request merged**